aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1529,5 +1879,15 @@ TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("_quantized::conv_transpose2d"),  QConvInt8<2, false>::run);
 }
 
+TORCH_LIBRARY_IMPL(onednn, MkldnnCPU, m) {
```

Comment:
are you planning to change MkldnnCPU to OnednnCPU in the future?

Reply:
yean, I think we may need to change the dispatchkey from `MkldnnCPU` to `OneDNNCPU` in the further. It's not specific for quantization but also for other data type or operations registered in `MkldnnCPU` dispatchkey.